smartsim/entity/model.py

```diff
@@ -152,12 +154,11 @@ def attach_generator_files(
 
     def colocate_db(self, *args: t.Any, **kwargs: t.Any) -> None:
         """An alias for ``Model.colocate_db_tcp``"""
-        warnings.warn(
+        logger.warning(
```

Comment:
is there a reason for the change from `warnings` to `logger` here? Only reason I ask is that in the past all deprecations were done via `warnings`

Reply:
I think it's because I didn't see any other usage of warnings in the codebase and got self-conscious. Agree that `warnings` is better.